app/views/tags/show.html.slim

```diff
@@ -6,31 +6,10 @@
         h6.muted Collection /
         h2.my-3 = @tag.text        
         p.muted Created #{@tag.created_at.strftime("%b %d, %Y")}
-      .row.mt-4
       .col-md-9
-        ul#tab.nav.nav-tabs role="tablist"
-          li.nav-item
-            a#posts-tab.nav-link.active data-toggle="tab" href="#posts" role="tab" aria-controls="posts" aria-selected="true"
-              | Posts
-              span.pill.tab-count.ml-2 = @posts.count
-
-          li.nav-item
-            a#citations-tab.nav-link data-toggle="tab" href="#citations" role="tab" aria-controls="citations" aria-selected="false"
-              | Citations
-              span.pill.tab-count.ml-2 = @generated_posts.count
-
-        .tab-content#tabContent.my-5
+        .tab-content#tabContent
           #posts.tab-pane.fade.show.active role="tabpanel" aria-labelledby="posts-tab"
-            - if @posts.empty?
-              p.muted.my-5 No posts have been added to this collection yet.
-            - @posts.each do |post|
-              = render 'posts/post', post: post
-
-            = will_paginate @posts, param_name: :posts_page, anchor: "posts"
-
-          #citations.tab-pane.fade role="tabpanel" aria-labelledby="citations-tab"
-            - @generated_posts.each do |post|
-              = render 'posts/post', post: post
-
-            = will_paginate @generated_posts, param_name: :citations_page, anchor: "citations"
-
+            = react_component "PostsContainer",
+              tag: @tag,
+              postsCount: @posts.count,
```

Comment:
just fyi https://medium.com/@craigsheen/count-vs-length-in-rails-4308e83f6292

Reply:
note to self: when in doubt use `.size`